hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2863,4 +2863,23 @@
     </description>
   </property>
 
+
+  <property>
+    <name>ozone.service.shutdown.timeout</name>
+    <tag>OZONE, SCM, OM, DATANODE</tag>
+    <value>60s</value>
+    <description>
+      Timeout to wait for each shutdown operation to complete.
+      If a hook takes longer than this time to complete, it will be interrupted,
+      so the service will shutdown. This allows the service shutdown
+      to recover from a blocked operation.
+      Some shutdown hooks may need more time than this, for example when DN
```

Comment:
" for example when DN" seems to be incomplete.

Reply:
Removed this.